lib/rust/ensogl/pack/js/src/runner/index.ts

```diff
@@ -214,7 +281,7 @@ export class App {
         })
         if (unrecognized.length > 0) {
             logger.error(`Unrecognized configuration parameters: ${unrecognized.join(', ')}.`)
-            this.showConfigOptions(unrecognized)
+            showConfigOptions(this.config, unrecognized)
```

Comment:
Actually, all the code above seems very similar to what EnsoGL pack does. Instead of doing all the above, could we just call function from EnsoGL pack that checks the amount of unrecognized things, displays the config options if there are unrecognized flags, etc? I'd prefer to keep as much logic as possible in one place without duplicating it.

Reply:
that code is in `ensogl/pack`
the lines in `content/src/index.ts` are:
```js
        const unrecognized = contentConfig.OPTIONS.loadAll([app.urlParams()])
        if (unrecognized.length > 0) {
            app.showConfigOptions(contentConfig.OPTIONS, unrecognized)
        }
```
i think this is a reasonable amount of logic to have here, not sure if it can be reduced?
note that one is calling a method on `Group`, the other is calling a top-level function